navis/meshes/b3d.py

```diff
@@ -37,6 +37,16 @@ def simplify_mesh_blender(x, F, inplace=False):
     simp
                 Simplified mesh object.
 
+    Examples
+    --------
+    >>> import navis
+    >>> n = navis.example_neurons(1, kind="mesh")
+    >>> n_sm = simplify_mesh_blender(n,
```

Comment:
I think it's a good idea to provide some example here. Currently, this will cause an error during doctests because Blender is not installed on the Github worker. 

You can either edit `test-package.yml` to install Blender before running pytest, or add the `# doctest: +SKIP` directive to exclude this line from the tests (see other such examples throughout the codebase). In theory, the former should be pretty straight forward: `wget` the linux distribution, unzip somewhere and add to `PATH`. In practice, Github Actions can be a pain in the neck to debug.

Reply:
Thanks, that's a great idea.

I had issue with the current installation with some mesa related libraries and removed `graphviz-dev` and `mesa-vulkan-drivers` from my `test-package.yml`. I also added the latest blender-LTS via `moguri/setup-blender@v1` action.

I did not see the tests finish, but in at least one of the runs I saw the test from b3d pass. Let me know if this could work for you?

Reply:
I updated the solution to keeping the ubuntu libraries as they were before and installing the blender directly via wget, not relying on an external and potentially unnecessary complex action. The `simplify_mesh_blender` seems to pass more reliably now, the comparison between basic types and numpy types still fails, so the overall tests currently still fail.

Let me know if I should change anything else?